tests/test_main.py

```diff
@@ -136,3 +141,27 @@ def test_verify_different_returncode_127(fake_process, caplog):
     assert "Failed to run command" in caplog.text
     assert fake_process.call_count(["meld", fake_process.any()]) == 1
     assert fake_process.call_count(["pycharm", fake_process.any()]) == 1
+
+
+def test_hello_default(monkeypatch, path):
+    monkeypatch.setattr("pytest_approval.main.AUTO_APPROVE", True)
+    assert not path.exists()
+    assert verify("new content")
+    assert path.exists()
+
+
+pytest_plugins = ["pytester"]
+
+def test_auto_approval(pytester):
+    pytester.makepyfile(
+        """
+        from pytest_approval import verify
+        def test_hello_default(path):
+            assert verify(path)
+    """
+    )
+
+    # run all tests with pytest
+    result = pytester.runpytest("--auto-approve")
+
+    result.assert_outcomes(passed=1)
```

Comment:
Can we factor out pytester tests to another test module (E.g. `test_pytest_plugin.py`)?

Reply:
i removed it, as it was already tested in the other test